notebooks/MAST/m_dwarf_flares_hst/m_dwarf_flares_hst.ipynb

```diff
@@ -0,0 +1,1412 @@
+{
```

Comment:
<p>This is so cool! It may be useful to point out ahead of time, e.g. when the table is loaded, which columns will be of interest to us for the purposes of the tutorial. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>